metal3-scaling-experiments/fake-controller.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
I won't repeat the same notes as from above script, they seem to all apply here as well.